content/en/docs/concepts/configuration/secret.md

```diff
@@ -854,6 +854,43 @@ start until all the Pod's volumes are mounted.
 
 ## Use cases
 
+### Use-Case: As container environment variables
+
+Create a secret
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  username: YWRtaW4=
+  password: MWYyZDFlMmU2N2Rm
```

Comment:
If these are to be environment variables how about making the names upper case?

Alternatively, _reuse_ the Secret manifest from https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#create-a-secret

Reply:
changed thx !